src/NewTools-Playground/StPlaygroundBindingPresenter.class.st

```diff
@@ -0,0 +1,102 @@
+Class {
+	#name : 'StPlaygroundBindingPresenter',
+	#superclass : 'StPresenter',
+	#instVars : [
+		'binding',
+		'parent',
+		'bindingName',
+		'labelName',
+		'bindingValue',
+		'button',
+		'buttonClose',
+		'interactionModel'
+	],
+	#category : 'NewTools-Playground-View',
+	#package : 'NewTools-Playground',
+	#tag : 'View'
+}
+
+{ #category : 'accessing' }
+StPlaygroundBindingPresenter class >> model: anModelPresenter [
+	^ self basicNew 
+		interactionModel: anModelPresenter parent interactionModel;
+		setModelBeforeInitialization: anModelPresenter;
+		initialize
+]
+
+{ #category : 'actions' }
+StPlaygroundBindingPresenter >> buttonAction [
+	self subclassResponsibility.
+]
+
+{ #category : 'layout' }
+StPlaygroundBindingPresenter >> defaultLayout [
+
+	^ SpBoxLayout newTopToBottom 
+			add: (SpBoxLayout new
+				   add: (SpBoxLayout newLeftToRight hAlignCenter add: labelName ; yourself) ;
+				   add: bindingName; 
+				   yourself) withConstraints: [:contrain | contrain width: 75; height: 60; padding: 10];
+		 	add: bindingValue;
+			add: (SpBoxLayout newLeftToRight hAlignCenter ;
+						add: button; 
+						add: buttonClose;
+						yourself) withConstraints: [ :contrain | contrain width: 70; height: 30 ];
+			
+		   yourself
```

Comment:
Ideally, #withContraints: should be avoided. Check with Esteban if we can avoid it here